src/librustc_mir/transform/check_unsafety.rs

```diff
@@ -1,15 +1,14 @@
-use rustc_data_structures::fx::FxHashSet;
-
-use rustc::hir;
-use rustc::hir::def_id::DefId;
-use rustc::hir::Node;
+use rustc::hir::intravisit;
```

Comment:
Hmm, shouldn’t this be `rustc_hir::intravisit`?

Reply:
Double-checked, its in `rustc::hir` for now still. Weird that we still have `rustc_hir` and `rustc::hir`.

Reply:
"Under construction" :P

`intravisit` depends on the HIR `Map`, which depends on query stuff, so we couldn't move it.